errors.go

```diff
@@ -43,6 +46,31 @@ func (e ErrInvalidInput) Error() string {
 	return e.choseErrString()
 }
 
+// ErrMissingEnvironmentVariable is the error when environment variable is required
+// in a particular situation but not provided by the user
+type ErrMissingEnvironmentVariable struct {
+	BaseError
+	EnvironmentVariable string
+}
+
+func (e ErrMissingEnvironmentVariable) Error() string {
+	e.DefaultErrString = fmt.Sprintf("Missing environment variable [%s]", e.EnvironmentVariable)
+	return e.choseErrString()
+}
+
+// ErrMissingAnyoneOfEnvironmentVariables is the error when anyone of the environment variables
+// is required in a particular situation but not provided by the user
+type ErrMissingAnyoneOfEnvironmentVariables struct {
+	BaseError
+	EnvironmentVariables []string
+}
+
+func (e ErrMissingAnyoneOfEnvironmentVariables) Error() string {
+	e.DefaultErrString = fmt.Sprintf("Missing anyone of environment variables [%s]",
+		strings.Join(e.EnvironmentVariables, ", "))
```

Comment:
For the error message, how about:

"Missing one of the following environment variables"

Also, if you want to split this on multiple lines, maybe follow the same format as line 86:

```go
e.DefaultErrString = fmt.Sprintf(
        "Missing one of the following environment variables [%s]",
        strings.Join(e.EnvironmentVariables, ", "))
```